src/create-node-middleware.ts

```diff
@@ -11,6 +11,6 @@ export function createNodeMiddleware(
   probot.load(appFn);
 
   return createWebhooksMiddleware(probot.webhooks, {
-    path: webhooksPath || defaultWebhooksPath,
+    path: webhooksPath || probot.webhookPath || defaultWebhooksPath,
```

Comment:
Setting the option at runtime is overriding any set environment variable. Should it be documented?